components/modal/leaflet.tsx

```diff
@@ -74,17 +82,27 @@ export const Leaflet = forwardRef<HTMLDivElement, PropsWithChildren<LeafletProps
                 dragElastic={{ top: 0, bottom: 1 }}
                 dragConstraints={{ top: 0, bottom: 0 }}
             >
-                <div className={`py-3 flex flex-col h-full z-40 ${height != 'full' ? 'bg-secondary-900 border-t border-secondary-500 rounded-t-2xl ' : ''}  pb-6`}>
-                    <div className='px-6 flex justify-between items-center'>
+                <div className={`py-3 flex flex-col h-full z-40 ${height != 'full' ? 'bg-secondary-900 border-t border-secondary-500 rounded-t-2xl ' : ''} pb-6`}>
+                    <div className='px-4 flex justify-between items-center'>
                         <div className="text-lg text-white font-semibold">
-                            <div>{title}</div>
+                            {userType != UserType.AuthenticatedUser && title == "Menu"
```

Comment:
This code shouldn't be in the leaflet, the header should be passed down from the menu componenet

Reply:
Leaflet should stay abstract, it shouldn't know anything about a user or email etc...

Reply:
Also this is bad `title == "Menu"`